aspnetcore/security/authorization/simple.md

```diff
@@ -73,13 +73,26 @@ This would allow only authenticated users to the `AccountController`, except for
 
 <a name="aarp"></a>
 
-## Authorize attribute and Razor Pages
+## Razor Pages authorization
 
-The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> can ***not*** be applied to Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. Consider using an ASP.NET Core MVC controller for pages with different authorization requirements for different handlers.
+You can require authentication on a Razor Page using the `<xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute>` attribute:
 
-The following two approaches can be used to apply authorization to Razor Page handler methods:
+```csharp
+[Authorize]
+public class AdminModel : PageModel
+{
+    public void OnGet()
+    {
+    }
+}
+```
+
+### Razor Pages handler method authorization
+
+The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> must be applied on the page instead of individual Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. The following approaches can be used if needed:
 
 * Use separate pages for page handlers requiring different authorization. Move shared content into one or more [partial views](xref:mvc/views/partial). When possible, this is the recommended approach.
+* Consider using an ASP.NET Core MVC controller for pages with different authorization requirements for different handlers.
```

Comment:
This is the best tip and shouldn't be removed.

Reply:
I disagree. Rewriting your Razor Pages to MVC seems like a more drastic change than simply splitting up a page. Also, the first option currently states `When possible, this is the recommended approach`. Could you explain why you think converting to MVC is the best option?

Reply:
> Could you explain why you think converting to MVC is the best option?

See https://github.com/dotnet/AspNetCore.Docs/pull/18879/files
That was my conclusion given it's so trivial. The ASP.NET core developers wrote the other alternatives and missed the simplest approach, so I PR'ed it and the developers agreed.

> Rewriting your Razor Pages to MVC seems like a more drastic change than simply splitting up a page.

You don't need to rewrite your RP to MVC, just the rare page that requires different authorization requirements for different handlers. Conversion is trivial. In practice, it's fairly rare and only a few pages at most.